src/main/zapHomeFiles/hud/drawer.html

```diff
@@ -67,6 +70,58 @@
         </div>
     </template>
 
+
+    <template id="websockets-template">
+        <div>
+            <table id="websockets-header" class="table drawer-header">
+                <thead>
+                    <tr>
+                        <th class="title time"> 
+                            {{ $t('message.websockets_message_field_time') }}
+                        </th>
+                        <th class="title direction"> 
+                            {{ $t('message.websockets_message_field_direction') }}
+                        </th>
+                        <th class="title opcode"> 
+                            {{ $t('message.websockets_message_field_opcode') }}
+                        </th>
+                        <th class="title length"> 
+                            {{ $t('message.websockets_message_field_bytes') }}
+                        </th>
+                        <th class="title payload"> 
+                            {{ $t('message.websockets_message_field_payload') }}
+                        </th>
+                        <th class="title filter"> 
+                            <span>
+                                <input id="websockets-filter" class="drawer-filter" v-model="filter" type="text" v-bind:class="{ 'error': isRegExError }" placeholder="Filter" />
+                            </span>
+                            <span>
+                                <input v-model="regexEnabled" type="checkbox" />
+                            </span>
+                            <span>
+                                {{ enableRegExText }}
+                            </span>
+                        </th>
+                        <th class="title hidden">
+                            <span>{{ websocketsItemsFilteredMessage }}</span>
+                        </th>
+                    </tr>
+                </thead>
+            </table>
+            <table id="websockets-messages" class="table table-striped table-hover table-scroll table-history drawer-messages" style="min-width: fit-content;">
+                <tbody>
+                    <tr v-for="message in filteredMessages" @click="messageSelected(message.id)" :id="'message-tr-'+message.id" class="message-tr">
+                        <td class="field time"> {{ message.time }} </td>
+                        <td class="field direction"> {{ message.direction }} </td>
+                        <td class="field opcode"> {{ message.opCode }} </td>
```

Comment:
These and following fields need some style tweaks to be shown under the respective column.